### DISCUSSION TITLE
Hotspot sync status not updating

### DISCUSSION THREAD

**Discussion by dutta-alankar** [helium/miner#1612] [Issues]
My hotspot is always online but its last synced status is showing a week ago. Its status isn't updating in the helium explorer.
 Meanwhile, I got my hotspot out of relay for the last week. Today, I got it relayed again. But my status remained relayed over the entire time in explorer. While other utilities like Hotspotty or Helium tracker showed my status as not relayed at that time.

**Comment by meowshka** [comment]
@dutta-alankar what version of the hotspot firmware do you have? Are you still experiencing this issue after switch to Light Hotspots?

Btw, we're moving all issues to Discussions board, so your original issue will be closed.

**Reply by dutta-alankar** [reply]
I remember that after about 10 days from getting my hotspot out of relay, the relay status on the explorer app was changed. With light hotspots, I think these issues are irrelevant.

**Comment by meowshka** [comment]
@dutta-alankar if you're on Android, please make sure you've update to the latest version 4.1.0. 
There's also a pending release for iOS that will address switch to the Light Hotspots and discrepancy between Explorer and app statuses.
